prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -270,6 +271,25 @@ object CustomFHIRFunctions {
         }
     }
 
+    /**
```

Comment:
Remove all of this stuff from your PR.  Make sure to go over all the changes in the PR, so you can identify anything that looks weird

Reply:
done